tutorials/algorithms/05_qaoa.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    result = npme.compute_minimum_eigenvalue(Operator(qubit_op.to_matrix()))
</pre><p>Might be a bit simpler to use </p><pre class="ql-syntax" spellcheck="false">result = npme.compute_minimum_eigenvalue(PauliSumOp(qubit_op))
</pre><p><code> </code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Ideally, following the resolution of [Terra#9091](https://github.com/Qiskit/qiskit-terra/issues/9091), we'd just use the `SparsePauliOp` itself. This is just a placeholder that avoids `PauliSumOp` due to its pending depreciation. I'm happy to change to `PauliSumOp` for now though.

Reply:
<p>Forgot about the deprecation! In general it seemed a bit too convoluted to first change to matrix and then define the operator from the matrix when there are simpler ways to do that. If there are other valid (i.e. not to-be-deprecated) classes that work without needing <code>to_matrix</code> it'd be better. If not, we can stick to what we have here.  </p>